contracts/UFragmentsPolicy.sol

```diff
@@ -37,11 +37,10 @@ contract UFragmentsPolicy is Ownable {
 
     UFragments public uFrags;
 
-    // Cpi oracle provides the current CPI-U, seasonally adjusted data,
-    // as a 18 decimal fixed point number
+    // Provides the current CPI, as an 18 decimal fixed point number.
```

Comment:
for line 47: Should we mention that it's the average of the last released 3 months of PCE price index?

Reply:
at line 47: for consistency let's use 18 decimal fixed point number. instead of DECIMALS fixed...

Reply:
Good catch, done.